service/service.go

```diff
@@ -201,27 +133,26 @@ func (s *Service) deploy() error {
 }
 
 // sendStatus sends a status message.
-func (s *Service) sendStatus(message string, typ DStatusType) {
-	if s.statuses != nil {
-		s.statuses <- DeployStatus{
+func (s *Service) sendStatus(statuses chan DeployStatus, message string, typ DStatusType) {
+	if statuses != nil {
+		statuses <- DeployStatus{
 			Message: message,
 			Type:    typ,
 		}
 	}
 }
 
 // closeStatus closes statuses chan.
-func (s *Service) closeStatus() {
-	if s.statuses != nil {
-		close(s.statuses)
+func (s *Service) closeStatus(statuses chan DeployStatus) {
```

Comment:
[NOT BLOCKING] Same here

Reply:
https://github.com/mesg-foundation/core/pull/855#discussion_r273485889